### DISCUSSION TITLE
Are there plans about transactions?

### DISCUSSION THREAD

**Discussion by isgj** [collerek/ormar#146] [Q&A] [answered]
I haven't seen any documentation regarding the use of `transactions`. I guess they are not yet supported.
Do you have any plan for this?

**Comment by collerek** [comment, marked as answer]
Transactions are supported with encode/databases `Database`: https://www.encode.io/databases/connections_and_transactions/

**Reply by isgj** [reply]
Wow, so all the queries inside that context will run in the transaction, cool
